### DISCUSSION TITLE
Set node version failing on M1

### DISCUSSION THREAD

**Discussion by Alvaro948** [codemagic-ci-cd/codemagic-docs#1485] [Q&A] [answered]
Hi, today, and without any change my side, the build has stopped working on the preparing machine step

```
== Use Node version hydrogen ==
> n hydrogen


Build failed :|
Unable to install Node version hydrogen
```

Build: 6412eb25793fbd7671d755e9

Changed "hydrogen" to "18.15.0", same issue: 6412ec8ac98660d52ad735bf
Changed "hydrogen" to "lts", same issue: 6412eda7793fbd7671d75a91

By removing node version from `environment` block at the `codemagic.yaml` the issue is now gone:

```yaml
environment:
      # node: hydrogen
```

Thanks!

**Comment by icarusdust** [comment, marked as answer]
Hi, @Alvaro948 it is an intermittent issue about the node server and people have been reporting it: https://github.com/nodejs/nodejs.org/issues/4495 As downloadable files are managed through Cloudflare, a bug could be causing this intermittent issue.
